src/App.js

```diff
@@ -1,24 +1,122 @@
-import logo from './logo.svg';
+/* import axios from 'axios';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store';
 import './App.css';
+import './styles/Navbar.scss';
+import Reservations from './components/Reservations';
+import Navbar from './components/navbar';
+import Home from './components/home';
+import DeleteRooms from './components/DeleteRooms';
+import AddReservation from './components/AddReservation';
+import Room from './components/Room';
+import LoginForm from './components/LoginForm';
+
+const token = localStorage.getItem('yourTokenKey'); // Retrieve the token from where you stored it
+
+if (token) {
+  axios.defaults.headers.common.authorization = `Bearer ${token}`;
+  console.log(axios.defaults.headers.common.authorization);
+}
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Provider store={store}>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="rooms/:id" element={<Room />} />
+          <Route path="/reservations" element={<Reservations />} />
+          <Route path="/addReservation" element={<AddReservation />} />
+          <Route path="/deleteRooms" element={<DeleteRooms />} />
+          <Route path="/loginForm" element={<LoginForm />} />
+        </Routes>
+      </Provider>
+    </BrowserRouter>
+  );
+}
+
+export default App;
+*/
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { AuthProvider } from './auth/AuthProvider';
+import store from './redux/store';
+import './App.css';
+import './styles/Navbar.scss';
+import Reservations from './components/Reservations';
+import Navbar from './components/navbar';
+import Home from './components/home';
+import DeleteRooms from './components/DeleteRooms';
+import AddReservation from './components/AddReservation';
+import Room from './components/Room';
+import LoginForm from './components/LoginForm';
+import PrivateRoute from './components/PrivateRoute';
+import SignupForm from './components/SignupForm';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice (I found 6):

![image](https://github.com/hernandanielzamora/Capstone-Front-End/assets/57421823/f143d212-9595-48ba-8614-2276ca29cac9)


You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.  

That way, your code will meet the latest standards. 💪